integrations/docker/images/chip-build-nrfconnect-platform/run.sh

```diff
@@ -1 +0,0 @@
-../../run.sh
```

Comment:
deleting this file and directory means that there's now no facility to do a "run" in the nrf-connect image.